webpack/strategies/docs.js

```diff
@@ -23,10 +23,10 @@ export default (config, options) => {
         noParse: /babel-core\/browser/,
         loaders: config.module.loaders
           .map(value => {
-            if (/\.js\/$/.test(value.test.toString())) {
+            if (/js/.test(value.test.toString())) {
```

Comment:
If `json` doesn't need `react_hot!`, then it seems that both `regex to test a regex`
have to be `/\.js\/$/`

Results from console:
`*` changed
`+` added

```sh
webpack.config
 { test: /\.js/,       loader: 'babel', exclude: /node_modules/ }

docs
*{ test: /\.js/,       loader: 'babel', exclude: /node_modules|Samples\.js/ },
+{ test: /Samples.js/, loader: 'transform?brfs!babel' }
+{ test: /\.json$/,    loader: 'json' }

docs-development
*{ test: /\.js/,       loader: 'react-hot!babel', exclude: /node_modules|Samples\.js/ },
*{ test: /Samples.js/, loader: 'react-hot!transform?brfs!babel' },
*{ test: /\.json$/,    loader: 'react-hot!json' },
```